src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -90,10 +91,12 @@ case class StateManager(depositDir: File, submitBase: File, easyHome: URL) exten
         // probably properly saved without toString but getSubmittedBagId would throw
         // ConversionException: 'bag-store.bag-id' doesn't map to a String object
         draftProps.setProperty(bagIdKey, bagStoreBagId.toString)
+        draftProps.setProperty(depositSourceKey, "API")
```

Comment:
No need to for an add-delete cycle. Just add this property in `DepositDir.createDepositProperties`.

By the way: the value `bag-store.bag-name` in that method seems to be obsolete. At submission we are now setting `bag-store.bag-id`.

Reply:
The value is independent of a submit attempt or any other state change. So the `StateManager` has nothing to do with this property. 

Add this property in `DepositDir.createDepositProperties`.